tests/unit-tests/config/config-manager.test.js

```diff
@@ -24,8 +24,14 @@ describe("ConfigManager", function(){
 	});
 
 	it("gets configuration", function(){
-		const config = ConfigManager.get_configuration("this.key");
-		assert.strictEqual(config instanceof Object, true);
```

Comment:
Because they brought no value to the test, I adjusted it to actually get the correct configuration.